src/main/java/com/microsoft/aad/msal4j/ClientApplicationBase.java

```diff
@@ -77,6 +75,10 @@
     @Getter
     private String applicationVersion;
 
+    @Accessors(fluent = true)
+    @Getter
+    private InstanceDiscoveryResponse instanceDiscoveryResponse;
```

Comment:
I would rename to AADInstanceDiscoveryResponse because it is only relevant for AAD authorities, 
so not for B2C or ADFS

Reply:
Ok will update